ContainerHandling/Get-LatestAlLanguageExtensionUrl.ps1

```diff
@@ -3,6 +3,10 @@
   Get the URL of the latest AL Language Extension from VS Code Marketplace
  .Description
   Get the URL of the latest AL Language Extension from VS Code Marketplace
+ .Parameter vsixFile

```

Comment:
There is a new function, which downloads and extracts the latest Vsix from marketplace - this function should not - it should "just" return the URL as it used to.
I am simply unable to explain to people why the vsixFile parameter is added to this GetUrl function.